modules/misc.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# Copyright 2022 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+function merge_mender_configs() {
+    declare -a ol_targets=()
+    local log="work/mergefiles/log"
+    local tmp="work/mergefiles/tmp"
```

Comment:
suggestion:

This is used as the temporary `mender.conf` file, right?
Maybe it should be named thus?

Also, why not use good old regular /tmp?

Reply:
with regards to /tmp, I think its best to keep everything in `work` as we do that just about everywhere else

Reply:
Why?

Also, it should be cleaned up after use.

Reply:
work is cleaned on EXIT signal